modules/run-vault/run-vault

```diff
@@ -104,6 +104,19 @@ function assert_is_installed {
   fi
 }
 
+function vault_version_at_least {
+  local readonly config_path=$1
+  local readonly ui_config=$2
+  local readonly VAULT_VERSION=$(/usr/local/bin/vault -v|awk '{print $2}'|tr -d v)
+
+  if [[ $(echo "$VAULT_VERSION 0.10.0" | tr " " "\n" |sort --version-sort| head -n 1) = 0.10.0 ]]
```

Comment:
I had to squint at this one even longer to understand that you were printing each version number on a line, sorting them, and making sure the earlier one is 0.10.0. Could you add a comment explaining this to make things easier for future maintainers?

Reply:
Ok, comments have been added and the get_vault_version function has been separated out.  And everything is still passing the automated tests.